evm/src/main/java/org/hyperledger/besu/evm/precompile/BigIntegerModularExponentiationPrecompiledContract.java

```diff
@@ -58,7 +58,7 @@ public class BigIntegerModularExponentiationPrecompiledContract
    * @param gasCalculator the gas calculator
    */
   public BigIntegerModularExponentiationPrecompiledContract(final GasCalculator gasCalculator) {
-    super("BigIntModExp", gasCalculator);
+    super("EXPMOD", gasCalculator);
```

Comment:
I wonder why you didn't choose MODEXP, which is the more widely used name.

Reply:
Will fix.